daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -551,6 +554,38 @@ codegenTests codegenDir damlTypes = testGroup "daml codegen" (
                         contents <- listDirectory outDir
                         assertBool "bindings were written" (not $ null contents)
 
+createDamlAppTests :: TestTree
+createDamlAppTests = testGroup "create-daml-app" [gettingStartedGuideTest | not isWindows]
```

Comment:
We did try running things manually on Windows right? Judging from the number of support requests that I see, things are going to blow up pretty badly if things don’t work on Windows.

Not quite sure what we do about this in the long-term, I guess we have to fix `rules_nodejs` :cry:

Reply:
I agree that we need to make sure this works on Windows. I don't have a Windows machine handy myself. Let's discuss this tomorrow morning.

I'm afraid fixing `rules_nodejs` is a bullet we have to bite. And that sucks a lot! 😿

Reply:
I just started testing, we get up to the `daml2ts` step which fails so I guess the good news is users “finish” quickstart on Windws in about 1 minute :tada: 

I know what the issue is and will fix it but I’m getting a bit tired so might have to wait until tomorrow.

Reply:
First fix in https://github.com/digital-asset/daml/pull/5391, now need to figure out how to build the npm packages on Linux and then copy them over to Windows. Very disappointed that Github does not have a vomitting emoji.

Reply:
Awesome, thanks @cocreature !